python/ray/tune/experimental/output.py

```diff
@@ -1089,3 +1090,16 @@ def _get_heartbeat(self, trials: List[Trial], force_full_output: bool = False):
 
     def _print_heartbeat(self, trials, *args, force: bool = False):
         print(self._get_heartbeat(trials, force_full_output=force))
+
+    def on_trial_result(
+        self,
+        iteration: int,
+        trials: List[Trial],
+        trial: Trial,
+        result: Dict,
+        **info,
+    ):
+        self._last_heartbeat_time = time.time()
```

Comment:
Why do we reset the heartbeat time here? Is it because we already print a metric table on trial result?